### DISCUSSION TITLE
We speak German and English / Wir sprechen Deutsch und Englisch

### DISCUSSION THREAD

**Discussion by buanet** [buanet/ioBroker.docker#241] [Announcements / Ankündigungen]
Hi, 
as you can see I changed the categories of the discussion section. So feel free to post your topics in German or English. 

Regards,
André

----------
Hi,
wie ihr sehen könnt habe ich die Titel der Kategorien in "Discussions" Bereich geändert. Postet also eure Themen gerne in Deutsch oder Englisch.  

MfG,
André

**Comment by McR1983** [comment]
Der oder die Discord Links sind nicht mehr gültig

https://discord.gg/kN7nhx6C

**Reply by buanet** [reply]
Geändert. Hier der Neue, hoffentlich dauerhaft gültige Link: https://discord.gg/5jGWNKnpZ8

MfG,
André
